### PR TITLE
add incident report for search outage

### PR DESCRIPTION
**Pull request by NuclearRedeye**

### CODE REVIEW

**Comment by NuclearRedeye** [comment]
@lsh-0 @giorgiosironi @nlisgo, can you have a look through the document and help flesh out the timeline please, and obviously add to any other sections where you can.

File: incidents/2020-05-20-seach-outage.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+- 09:42 Nathan looked at the GitHub issue, identied that it might be related to search but said he wouldn't be able to look into the problem until the afternoon.
+
+- 17:31 Incident Response channel on slack, message posted by Rowena about eToc being sent out with no content.
+
+- 17:40 Giorgio started investigation
+
+**2020-05-21**
+
+- 05:07 Luke S notes that the issue is now resolved
```
Comment (giorgiosironi) [inline incidents/2020-05-20-seach-outage.md]: https://github.com/elifesciences/issues/issues/5664#issuecomment-632756145 was a separate problem which I'd consider out of the scope of this incident.

File: incidents/2020-05-20-seach-outage.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- eToc emails sent out to subscribers with no content.
+
+MTTD: 96h (estimated)
+MTTR: 35h
+
+## Corrective Actions
+
+- Was it related to [this](https://github.com/elifesciences/tech-team/pull/23/files)
+- [Luke raised the following](https://github.com/elifesciences/issues/issues/5664)
+- [Investigate if log rotation needs to be enabled]()
```
Comment (giorgiosironi) [inline incidents/2020-05-20-seach-outage.md]: we usually do create issues for these, under the [`post-mortem` label](https://github.com/elifesciences/issues/labels/post-mortem)

File: incidents/2020-05-20-seach-outage.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# 2020-05-20 Search Outage
+
+**Incident Leader: Giorgio Sironi, Luke Skibinski**
+
+## Description
+
+Summary of the findings of the investigation/post-mortem.
```
Comment (giorgiosironi) [inline incidents/2020-05-20-seach-outage.md]: to be filled in?

File: incidents/2020-05-20-seach-outage.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# 2020-05-20 Search Outage
```
Comment (giorgiosironi) [inline incidents/2020-05-20-seach-outage.md]: to be more precise, search wasn't out of service but it stopped accepting writes

**Review by lsh-0 — APPROVED** [review]
there are some typos but the gist is there

**Comment by lsh-0** [comment]
I've already approved this.

**Pull request merged**